blocks/header/header.js

```diff
@@ -1,7 +1,486 @@
+import { createTag } from '../../scripts/scripts.js';
+import leaderboard from './leaderboard.js';
+
+const DEFAULT_NAV = '/golf-nav';
+
+const config = {
+  setNavTop: true,
+  searchExposed: false,
+  highlightHeader: true,
+  useFixedMenus: false, // Area header uses fixed menus
+  headerSelector: '.header',
+  openClass: 'is-open',
+  activeClass: 'is-Active',
+  searchOpenClass: 'has-open-search',
+  fixMenuOpen: 'has-FixedMenu',
+  navMenuSel: '[data-module="golf-mobile-nav"]',
+  navButtonSel: '[data-type="button-header-nav"]',
+  searchBoxSel: '[data-mobile-search-box]',
+  searchInputSel: '[data-type="search-input"]',
+  anyHeadingSel: 'h1, h2, h3, h4, h5, h6',
+
+};
+
+let roots = Array.from(document.querySelectorAll('li.nav-level-0'));
```

Comment:
Similar to CSS selector, can you add a common class prefix to all selectors ? eg `.header li.nav-level-0` so you don't query accidentally something you don't expect.